front/style/manager-space/roles/_referent.scss

```diff
@@ -1,6 +1,7 @@
 .referent {
+
     h2 {
-        font-family: $font-gillsans;
+        font-family: $font-roboto, $font-gillsans;
```

Comment:
Why put the 2 typefaces ? You can remove gillsans if you don't want it in there (roboto already has some fallback fonts).

Reply:
👆